docs/BasicCommunication/ActivateApp/index.md

```diff
@@ -31,13 +31,15 @@ This request may be sent:
 ### Behavior
 
 If `level` is `FULL` or `LIMITED`: 
+
 !!! must
 
   1. Activate the application on the HMI
-    * Display [UI.Show](../../UI/Show) related parameters associated with the named `appID` in the case they were previously requested within ignition cycle
-    * Display the corresponding template in the case one was previously requested by [UI.SetDisplayLayout](../../UI/SetDisplayLayout) for that application
-    * Apply [UI.SetGlobalProperties](../../UI/SetGlobalProperties) associated with the named `appID`, if any
-    * Apply [UI.AddSubMenu](../../UI/AddSubMenu) associated with the named `appID`, if any
+  * Display [UI.Show](../../UI/Show) related parameters associated with the named `appID` in the case they were previously requested within ignition cycle
+  * Display the corresponding template in the case one was previously requested by [UI.SetDisplayLayout](../../UI/SetDisplayLayout) for that application
+  * Apply [UI.SetGlobalProperties](../../UI/SetGlobalProperties) associated with the named `appID`, if any
+  * Apply [UI.AddSubMenu](../../UI/AddSubMenu) associated with the named `appID`, if any
+  * Display the web page view in the main window of the application, in case the application `AppHMIType` is `WEB_VIEW`
```

Comment:
```suggestion
```

The webpage view should be shown if the displayLayout is WEB_VIEW not only if the appHMIType is webView. This information is covered by:

>   * Display the corresponding template in the case one was previously requested by [UI.SetDisplayLayout](../../UI/SetDisplayLayout) for that application